tasks/support/gem_base64.rb

```diff
@@ -0,0 +1,23 @@
+require "base64"
+require "rubygems/package"
+
+class GemBase64
+
```

Comment:
surprised to see this in support.
what is the difference between `tasks/support` and `lib`?